This patch has been integrated into Glance 11.0.0 in Liberty but has
not yet been back-ported to Kilo.

commit fa30891cf659360207b71d9345666478d4554582
Author: Erno Kuvaja <jokke@usr.fi>
Date:   Tue Sep 15 14:57:29 2015 +0000

    Update Glance example configs to reflect Liberty
    
    Adding taskflow_executor_opts into the opts so they will be included.
    
    Closes-bug: #1496012
    
    Depends-On: I52ebf810f4699826baa2bdf91d28e24d902cf950
    Change-Id: I9c0988a70f691482258f5f3ba9a5cf5601a81ddf

diff --git a/glance/opts.py b/glance/opts.py
index 1542d13..a626a3b 100644
--- a/glance/opts.py
+++ b/glance/opts.py
@@ -25,6 +25,7 @@ import itertools
 
 import glance.api.middleware.context
 import glance.api.versions
+import glance.async.taskflow_executor
 import glance.common.config
 import glance.common.location_strategy
 import glance.common.location_strategy.store_type
@@ -51,6 +52,7 @@ _api_opts = [
         glance.common.wsgi.bind_opts,
         glance.common.wsgi.eventlet_opts,
         glance.common.wsgi.socket_opts,
+        glance.common.wsgi.profiler_opts,
         glance.image_cache.drivers.sqlite.sqlite_opts,
         glance.image_cache.image_cache_opts,
         glance.notifier.notifier_opts,
@@ -61,6 +63,8 @@ _api_opts = [
         glance.scrubber.scrubber_opts))),
     ('image_format', glance.common.config.image_format_opts),
     ('task', glance.common.config.task_opts),
+    ('taskflow_executor',
+     glance.async.taskflow_executor.taskflow_executor_opts),
     ('store_type_location_strategy',
      glance.common.location_strategy.store_type.store_type_opts),
     ('paste_deploy', glance.common.config.paste_deploy_opts)
